pyfolio/plotting.py

```diff
@@ -645,7 +645,7 @@ def show_perf_stats(returns, factor_returns=None, positions=None,
         perf_stats = pd.DataFrame(perf_stats_all, columns=['Backtest'])
 
     for column in perf_stats.columns:
-        for stat, value in perf_stats[column].iteritems():
+        for stat, value in perf_stats[column].items():
```

Comment:
I think this needs to stay as it's a Series, not a dict.

Reply:
cool